core/base-service/got.js

```diff
@@ -64,7 +64,8 @@ async function sendRequest(gotWrapper, url, options) {
   }
 }
 
-function fetchFactory(fetchLimitBytes) {
+const TEN_MB = 10485760
+function fetchFactory(fetchLimitBytes = TEN_MB) {
```

Comment:
There are a couple of places in this PR where we want to call this outside of the context of the `Server` object. This means we can't use the config/settings because we only parse them when we construct a `Server` object. For now I have decided to just give this param a default value and deal with it that way, but slightly more widely I'm thinking of looking at how we could decouple settings parsing from bootstrapping the server at some point. It is fringe, but another issue where it would be useful to read the config settings outside the context of the server object would be #6268 . Another option would be to say HTTP calls made outside the server are 'out of scope' for these settings. It is something for another day though. I'll continue to chew it over as I progress #4655